packages/blitz-shell/src/event.rs

```diff
@@ -26,6 +26,9 @@ pub enum BlitzEvent {
 
     /// An arbitary event from the Blitz embedder
     Embedder(Arc<dyn Any + Send + Sync>),
+
+    /// Navigate to another URL (triggered by e.g. clicking a link)
+    Navigate(String),
```

Comment:
Or we make this an URL here? I wasn't sure... can one navigate to something that's not an URL? The HTML spec is convoluted, I don't know even a 100th of the special cases...

Reply:
> can one navigate to something that's not an URL

Probably not on the web. But I'd be inclined to leave this as a `String` for now. Seems like it could be useful for people who want to make their own custom URL representations.